.buildkite/scripts/publish_docker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
is this script being used anywhere?

Reply:
It was part of the publish locally workflow. I've removed it along with the makefile commands.